v2/pkg/protocols/dns/operators.go

```diff
@@ -147,3 +149,14 @@ func traceToString(traceData *retryabledns.TraceData, withSteps bool) string {
 	}
 	return buffer.String()
 }
+
+func recordsKeyValue(resourceRecords []dns.RR) output.InternalEvent {
+	var oe = make(output.InternalEvent)
+	for _, resourceRecord := range resourceRecords {
```

Comment:
I think this will overwrite multiple records of the same type (for example `a` or `aaaa`) always with the latest value

Reply:
updated the code to add all the records